requirements.txt

```diff
@@ -2,3 +2,4 @@
 # Django and openEdx's edx-platform are the runtime for this app.
 
 google-api-python-client==1.6.2
+web-fragments==0.2.2
```

Comment:
Is `web-fragments` installed into devstack/fullstack?
Anyways, add it to `setup.py`